plugins/modules/cronvar.py

```diff
@@ -396,6 +396,8 @@ def main():
     old_value = cronvar.find_variable(name)
 
     if ensure_present:
+        if value == "":
```

Comment:
Right now there's an idempotency problem: if someone already used the module to set the value to an empty string on a platform where it worked, they will now see changed output. How about extending the condition a bit so that if the new value is the empty string, then it's only replaced by `""` if the old value is not the empty string?
```suggestion
        if value == "" and old_value != "":
```
This way the module will accept existing `FOO=` lines.

Reply:
Thanks for pointing this out!